UK_Power_Generation_Units_BMRS/Python/Code/ukpowergeneration/visualisation/index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<title>Digital Twin Example</title>
+	<meta charset="utf-8">
+
+	<!-- External CSS -->
+	<link href='https://api.tiles.mapbox.com/mapbox-gl-js/v2.3.0/mapbox-gl.css' rel='stylesheet' />
+	<link rel="stylesheet" href="//code.jquery.com/ui/1.13.0/themes/base/jquery-ui.css">
+
+	<!-- External JS -->
+	<script src="https://code.jquery.com/jquery-3.6.0.js"></script>
+  	<script src="https://code.jquery.com/ui/1.13.0/jquery-ui.js"></script>
+
+	<script src='https://api.mapbox.com/mapbox-gl-js/v2.3.0/mapbox-gl.js'></script>
+	<script src="https://cdn.jsdelivr.net/npm/moment@2.29.1"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/3.5.1/chart.js"></script>
+	<script src="https://cdn.jsdelivr.net/npm/chartjs-adapter-moment@1.0.0"></script>
+	<script src='https://unpkg.com/@turf/turf@6/turf.min.js'></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/suncalc/1.8.0/suncalc.min.js"></script>
+
+	<!--
+		The following visualisiation framework files are currently stored locally (as
+		this is	easier during development and initial testing. Once it's ready for
+		release, these files will be hosted remotely).  
+	-->
+	
+	<!-- Framework CSS -->
+	<link href='./css/dtvf.css' rel='stylesheet' />
+	<link href="./css/jsonview.bundle.css" rel="stylesheet"> 
+
+	<!-- Framework JS -->
+	<script src='./js/manager.js'></script>
+	<script src='./js/data_registry.js'></script>
+	<script src='./js/source_handler.js'></script>
+	<script src='./js/layer_handler.js'></script>
+	<script src='./js/control_handler.js'></script>
+	<script src='./js/panel_handler.js'></script>
+	<script src='./js/interaction_handler.js'></script>
+	<script src='./js/timeseries_handler.js'></script>
+	<script src='./js/icon_handler.js'></script>
+	<script src='./js/jsonview.bundle.js'></script>
```

Comment:
```suggestion
	        <script src="./twa-vf/twa-vf.min.js" charset="utf-8"></script>

```

remove this and replace with twavf image